### PR TITLE
Remove NUnit 2 Support

### LINKED ISSUE #47: Remove NUnit 2
For the next release, we will be removing NUnit 2 support.

### PR DESCRIPTION
**Pull request by rprouse**
This removes NUnit 2 support and also cleans the project up and gets it ready for release. This is a new identifier, @OsirisTerje do you want to release this under your account once merged?

Fixes #47

### CODE REVIEW

File: CreateUnitTests.NUnit/NUnit3FrameworkProvider.cs [hunk 1]
```diff
@@ -1,25 +1,4 @@
...
-// 
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
-// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
-// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
-// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-// ***********************************************************************
+// Copyright (c) Terje Sandstrom and Contributors. MIT License - see LICENSE.txt
```
Comment (rprouse) [inline CreateUnitTests.NUnit/NUnit3FrameworkProvider.cs:1]: Switched to the short copyright in all files like other projects

File: README.md [hunk 2]
```diff
@@ -28,6 +31,6 @@ For VS 2015:
 
 ## Documentation
 
-Documentation and a Quick Start Guide can be found on the [NUnit Documentation Wiki](https://github.com/nunit/docs/wiki/Visual-Studio-Test-Generator).
+Documentation and a Quick Start Guide can be found on the [NUnit Documentation Wiki](https://docs.nunit.org/articles/vs-test-generator/TestGenerator-Installation.html).
```
Comment (rprouse) [inline README.md:34]: Documentation and release notes have moved.

File: TestGeneration.Extensions.NUnit/TestGeneration.Extensions.NUnit.csproj [hunk 3]
```diff
@@ -31,6 +31,12 @@
     <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Common debugging support -->
+    <StartAction>Program</StartAction>
+    <StartProgram>$(DevEnvDir)\devenv.exe</StartProgram>
+    <StartArguments>/rootsuffix Exp</StartArguments>
+  </PropertyGroup>
```
Comment (rprouse) [inline TestGeneration.Extensions.NUnit/TestGeneration.Extensions.NUnit.csproj:39]: Launch into a test instance of Visual Studio for testing and debugging.

**Review by OsirisTerje — APPROVED** [review]
LGTM

**Pull request merged**